src/backend/src/services/reimbursement-requests.services.ts

```diff
@@ -122,6 +122,49 @@ export default class ReimbursementRequestService {
     return createdReimbursementRequest;
   }
 
+  /**
+   * Function to reimburse a user for their expenses.
+   *
+   * @param recipientId the user who will be receiving the reimbursement
+   * @param amount the amount to be reimbursed
+   * @param submitter the person performing the reimbursement
+   * @returns the created reimbursement
+   */
+  static async reimburseUser(recipientId: number, amount: number, submitter: UserWithTeam): Promise<Reimbursement> {
+    await validateUserIsPartOfFinanceTeam(submitter);
```

Comment:
I know this is what the ticket said, but according to the epic https://github.com/Northeastern-Electric-Racing/FinishLine/issues/493 users are gonna report their own reimbursements, so you should check that the recipientId is the same as the submitter instead of checking that they're on the finance team